integrations/java-iceberg-1.5/openhouse-java-itest/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
```

Comment:
I am wondering if we should layout directories like

integrations/java/iceberg-1.2/..
integrations/java/iceberg-1.5/..

Reply:
➕  for maintaining backward compatibility, you can use `settings.gradle`'s module renaming feature.